config/locales/titles/en.yml

```
@@ -33,7 +33,7 @@ en:
       cancellation_prompt: Cancel identity verification
       cancelled: Identity verification is cancelled
       come_back_soon: Come back soon
-      enter_security_code: Enter your security code
+      enter_security_code: Enter your one-time code
```

If we're changing the text from "security code" to "one-time code", I think it'd be clearer if we updated the key accordingly.

```suggestion
      enter_one_time_code: Enter your one-time code
```

---

Sorry, I hit merge too eagerly. I agree with this & will follow up.